crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1409,9 +1409,13 @@ where
             validity_threshold = validity,
             ?timeout_after_quorum,
             ?cert_ref,
-            ?cert_bytes,
             "Broadcasting certificate to authorities"
         );
+        debug!(
```

Comment:
sam thing about log level here. 

Can we make a TODO explaining why we log this (for this incident)?

Reply:
This is not printed because anything goes wrong, so waybe we should use info instead of warn here?